common/setups/hybrid/gmm_system.py

```diff
@@ -234,8 +249,8 @@ def cart_and_lda(
             generalized_eigenvalue_args=generalized_eigenvalue_args,
         )
         self.jobs[corpus]["cart_and_lda_{}_{}".format(corpus, name)] = cart_lda
-        self.lda_matrices["{}_{}".format(corpus, name)] = cart_lda.last_lda_matrix
-        self.cart_trees["{}_{}".format(corpus, name)] = cart_lda.last_cart_tree
+        self.lda_matrices[corpus][name] = cart_lda.last_lda_matrix
```

Comment:
This seems to be a very reasonable change, but can we also change `corpus` to `corpus_key` everywhere this is a string? This would distinguish this much better from other code, where `corpus` is an actual corpus.

Reply:
I agree. This is also inconsistent in `i6_core`.
One way to define this would be:
`corpus_key` -> `str` (basically corpus identifier, `corpus_name` would also be an option)
`corpus_object` -> `CorpusObject`
`corpus_file` -> `str` (would be filepath)
`corpus` -> `Path`
I am open for suggestions on this.
@curufinwe your opinion?